src/WhatToCook.WebApp/ClientApp/src/app/meal-planner/shopping-list/shopping-list.component.html

```diff
@@ -0,0 +1,11 @@
+<label>Choose a Meal Plan:</label>
+<select [(ngModel)]="selectedMealPlanId" (change)="onMealPlanSelected()">
+    <option *ngFor="let plan of mealPlans" [value]="plan.id">{{plan.name}}</option>
+</select>
+
+<div *ngFor="let item of shoppingListByDate">
```

Comment:
it is missing functionality - to view ingredients day by day as mentioned in the issue. Maybe add some button/switch or something to allow to change view from one to another. Also, I think we should add some possibility to copy it via button click to the clipboard